app/controllers/concerns/foreman/controller/parameters/template_params.rb

```diff
@@ -34,7 +34,21 @@ def ui_template_export_params
         end
 
         def template_import_params
-          add_taxonomy_params(base_import_params(:none))
+          transform_lock_param add_taxonomy_params(base_import_params(:none))
+        end
+
+        def transform_lock_param(params)
+          lock = params[:lock]
+          return params unless lock
```

Comment:
If we support both string `"true"`/`"false"` and boolean `true`/`false` (as I see below) then probably we should change it to `return params if lock.nil?`, shouldn't we?

Reply:
Good catch.